databases/databases/project_sql_zoo.md

```diff
@@ -5,14 +5,15 @@ SQL Zoo is one of the few resources online that actually lets you build and run
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-  1. Go to [SQL Zoo](https://sqlzoo.net/) and do Tutorials 0-9 listed under the "Tutorial Section" and the quizzes listed at the end of each.  The first is the ["Select" tutorial](https://sqlzoo.net/wiki/SELECT_basics).  Make sure the dropdown on the upper right of the main page for "Engine" says "MySQL" (the default).  Large results will be cut off and not all rows or columns shown, so the "answers" may not look 100% correct.
-  2. Before you move on, we would like your feedback [here](https://docs.google.com/forms/d/e/1FAIpQLSenvMG6WFbOOEap_biQOwqfbH-j-xsf5Eyv4ir2Rx5FsYSecQ/viewform?usp=sf_link). Getting user(you) feedback is important so we can continue to improve the curriculum and get an idea of your experience.
+  1.  Go to [SQL Zoo](https://sqlzoo.net/) and do Tutorials 0-9 listed under the "Tutorial Section" and the quizzes listed at the end of each.  The first is the ["Select" tutorial](https://sqlzoo.net/wiki/SELECT_basics).
+      -   Make sure the dropdown on the upper right of the main page for "Engine" says "MySQL" (the default).  Large results will be cut off and not all rows or columns shown, so the "answers" may not look 100% correct.
+  2.  Before you move on, we would like your feedback - [please fill this form](https://docs.google.com/forms/d/e/1FAIpQLSenvMG6WFbOOEap_biQOwqfbH-j-xsf5Eyv4ir2Rx5FsYSecQ/viewform?usp=sf_link). Getting user (you) feedback is important so we can continue to improve the curriculum and get an idea of your experience.
```

Comment:
@ChargrilledChook 
I know you haven't mentioned this bit but I changed the prompt for filling out the form to be more in like with how linking should be done, I know that there are other cases like this and could make them match this - does it make sense?

Reply:
@scheals I compared and don't have strong feelings about it either way. Your edit probably improves readability slightly, but it's not major.

Contributor's choice :slightly_smiling_face:

Reply:
Then I'll keep my edit :D